src/components/Thanks/AnimatedStars.vue

```diff
@@ -1,73 +1,105 @@
 <!-- eslint-disable max-len -->
 <template>
 	<div>
-		<svg
-			class="svg first-svg"
-			width="24"
-			height="24"
-			viewBox="0 0 24 24"
-			fill="none"
-			xmlns="http://www.w3.org/2000/svg"
-		>
-			<path
-				d="M15.6562 9.56756C10.5937 9.56756 9.32812 5.18919 9.32812 3C9.32812 5.18919 8.0625 9.56756 3 9.56756C8.0625 9.56756 9.32812 14.1892 9.32812 16.5C9.32812 14.1892 10.5937 9.56756 15.6562 9.56756Z"
-				fill="#F4B539"
-				stroke="#F4B539"
-				stroke-width="1.5"
-				stroke-linecap="round"
-				stroke-linejoin="round"
-			/>
-			<path
-				d="M18.75 16.5V21"
-				stroke="#F4B539"
-				stroke-width="1.5"
-				stroke-linecap="round"
-			/>
-			<path
-				d="M21 18.75L16.5 18.75"
-				stroke="#F4B539"
-				stroke-width="1.5"
-				stroke-linecap="round"
-			/>
-			<circle cx="19.875" cy="4.125" r="1.125" fill="#F4B539" />
-		</svg>
+		<template v-if="badgeCompleted">
```

Comment:
I think it might be worth creating a new component just for the badge experience, since having MyKiva use a Thanks page component is a little confusing for maintenance reasons.